metashare/api/models.py

```diff
@@ -312,6 +395,17 @@ def finalize_create_pr(self, error=None):
         else:
             self.notify_error(error)
 
+    def add_commits(self, commits):
+        commit_set = self.project.commit_set | self.project.repository.commit_set
+        self.commits += [
+            gh.normalize_commit(c) for c in commits if c["id"] not in commit_set
+        ]
```

Comment:
My understanding is that we no longer want to exclude commits that are already in the parent. We want to include all commits on this branch, including ones that have been merged in.